client/scripts/build-stats.js

```diff
@@ -0,0 +1,75 @@
+import { writeFileSync } from 'node:fs'
+import { join } from 'node:path'
+import { fileURLToPath } from 'node:url'
+
+let GITHUB_REPO = 'browserslist/browserslist'
+let NPM_PACKAGE = 'browserslist'
+
+let ROOT = join(fileURLToPath(import.meta.url), '..', '..')
```

Comment:
We can use `import.meta.dirname` for that:

```js
const ROOT = join(import.meta.dirname, '..')
```

Also it should be `const` (and next one too.